packages/govuk-frontend/src/govuk/components/task-list/_index.scss

```diff
@@ -0,0 +1,72 @@
+@include govuk-exports("govuk/component/task-list") {
```

Comment:
Task names that aren't links and statuses that aren't tags do not currently have a text colour set on them, which means they're using the default user agent foreground colour.

This is usually black – which would mean that these elements are `#000000` rather than GOV.UK's `#0b0c0c`, but the default text colour can be overridden by the user:

![Screenshot 2023-06-08 at 11 50 52](https://github.com/alphagov/govuk-frontend/assets/121939/8a1be993-49af-4c16-833a-77ab52024986)

I think this would fail WCAG based on [Failure 24](https://www.w3.org/TR/WCAG20-TECHS/F24.html).

The easiest fix is either to `@include govuk-text-colour;` on both `.govuk-task-list__task-name-and-hint` and `govuk-task-list__status`, or on the root `.govuk-task-list`.

Reply:
Thanks - added `@include govuk-text-colour;` to both `govuk-task-list__task-name-and-hint` and `govuk-task-list__status`.